scripts/start.sh

```diff
@@ -100,15 +100,17 @@ rm assets/constants.js.bak
 
 # Set up a local dev instance.
 # TODO(sll): do this in a new shell.
-echo Starting GAE development server
 # To turn emailing on, add the option '--enable_sendmail=yes' and change the relevant
 # settings in feconf.py. Be careful with this -- you do not want to spam people
 # accidentally!
 
 if ! [[ "$FORCE_PROD_MODE" == "True" ]]; then
   ($NODE_PATH/bin/node $NODE_MODULE_DIR/gulp/bin/gulp.js watch)&
+  echo Compiling webpack...
+  $NODE_MODULE_DIR/webpack/bin/webpack.js --config webpack.dev.config.ts
```

Comment:
Just to check, should this be done every time the server starts, or are there cases in which we can omit it?

Also why do we not do any compilation in prod mode? Do things work OK when running the local server in prod (minified) mode?

Reply:
I think it is better to run it every time the server starts.

In prod mode, the webpack is called from `scripts/build.py`.

Reply:
Oh ok. I think maybe it might be worth mentioning that here in a comment, it's a natural question for a dev to wonder.